qa/zcash/smoke_tests.py

```diff
@@ -209,15 +210,15 @@ def z_sendmany(results, case, zcash, from_addr, recipients):
 def check_z_sendmany(results, case, zcash, from_addr, recipients):
     txid = z_sendmany(results, case, zcash, from_addr, recipients)
     if txid is None:
-        return decimal.Decimal('0')
+        return [decimal.Decimal('0')]
```

Comment:
This method is only ever called with one recipient. We should either change it to take in only the one recipient and not return a list ,or keep it general and return a list the same size as recipients here.